replay-protected-sighash.md

```diff
@@ -1,10 +1,10 @@
-# BUIP-HF Digest for replay protected signature verification accross hard forks
+# BUIP-HF Digest for replay protected signature verification across hard forks
 
 ## Abstract
 
 This document describes proposed requirements and design for a reusable signing mechanism ensuring replay protection in the event of a hard fork. It provides a way for users to create transactions which are invalid on forks lacking support for the mechanism and a fork-specific ID.
 
-The proposed digest algorithm is adapted from [BIP143](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki) as it minimizes redundant data hashing in verification, covers the input value by the signature [1] and is already implemented in a wide variety of applications.
+The proposed digest algorithm is adapted from [BIP143](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki) as it minimizes redundant data hashing in verification, covers the input value by the [signature][1] and is already implemented in a wide variety of applications.
```

Comment:
The "signature" link now goes to the "Motivation" section of BIP143, which is not optimal. It might be better to introduce another reference to go to top level of BIP143.